externs/olx.js

```diff
@@ -1680,6 +1681,15 @@ olx.format.GeoJSONOptions.prototype.defaultDataProjection;
 
 
 /**
+ * Projection for features read or written by the format.  Options passed to
+ * read or write methods will take precedence.
+ * @type {ol.ProjectionLike}
+ * @api stable
```

Comment:
I think it makes sense to call this stable.  I'll merge as is, and if there is disagreement, we can change it before the release.